src/cljs/commiteth/bounties.cljs

```diff
@@ -62,6 +62,8 @@
      :value     (:current-val opts)
      :on-change (when-let [f (:on-change-val opts)]
                   #(-> % .-target .-value int f))
+     :on-mouse-up (when-let [f (:on-change-val opts)]
```

Comment:
Fix for IE. Need to do the same as in on-change event. Otherwise input range value will not be populated to the "Value" field. See similar [issue in grommet project](https://github.com/grommet/grommet/issues/241).